es-app/src/views/gamelist/BasicGameListView.cpp

```diff
@@ -111,13 +111,13 @@ void BasicGameListView::remove(FileData *game, bool deleteFile)
 	{
 		std::vector<FileData*> siblings = parent->getChildrenListToDisplay();
 		auto gameIter = std::find(siblings.cbegin(), siblings.cend(), game);
-		int gamePos = (int)std::distance(siblings.cbegin(), gameIter);
+		unsigned int gamePos = (int)std::distance(siblings.cbegin(), gameIter);
```

Comment:
I have no real comment here, just that it took me (too) long to grok. Maybe it's because I am used to read Exceptions-first, default behaviour-last? I dont know. but never mind, it's fine as is.